src/cli.ts

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env node
+import fs from "node:fs";
+import path from "node:path";
+import simpleGit from "simple-git";
+import { type DiffResult, generateDiffs } from "./diff.js";
+
+/**
+ * CLI options for pickdiff.
+ */
+interface CliOptions {
+  repoPath: string;
+  startCommit: string;
+  endCommit: string;
+  files: string[];
+  contextLines: number;
+  output: "markdown" | "unified";
```

Comment:
What does "unified" output mean? Without using the CLI, we have the options for viewing the diff in HTML (on the page that opens when you run `npm start`) or exporting via markdown.